storage/kvstore_test.go

```diff
@@ -831,3 +815,64 @@ func (i *fakeIndex) Compact(rev int64) map[revision]struct{} {
 	return <-i.indexCompactRespc
 }
 func (i *fakeIndex) Equal(b index) bool { return false }
+
+func createMultiBytesSlice(size, bytesSize int) [][]byte {
+	p := make(map[string]struct{})
+	rs := [][]byte{}
+	for len(p) != size {
+		mrand.Seed(time.Now().UnixNano())
+		v := make([]byte, bytesSize)
+		if _, err := rand.Read(v); err != nil {
+			panic(err)
+		}
+		if _, ok := p[string(v)]; !ok {
+			rs = append(rs, v)
+			p[string(v)] = struct{}{}
+		} else {
+			continue
+		}
+	}
+	return rs
+}
+
+func TestTxnPut(t *testing.T) {
```

Comment:
I am actually not quite sure about what we want to test here. From the implementation, I can see we try to put foo->bar n times and rand key,value n times. What is the goal of this test? Ensure what?